File: lib/objetos/TipoUsuario.dart
class TipoUsuario {
 final secuencial;
 final descripcion;
 final estaActivo;

 TipoUsuario({
   this.secuencial,
   required this.descripcion,
   required this.estaActivo,
 });

 factory TipoUsuario.fromJson(Map<String, dynamic> json) {
   return TipoUsuario(
     secuencial: json['secuencial'],
     descripcion: json['descripcion'],
     estaActivo: json['estaActivo'],
   );
 }

 Map<String, dynamic> toJson() {
   return {
     'secuencial': secuencial,
     'descripcion': descripcion,
     'estaActivo': estaActivo,
   };
 }
}

File: src/models/TipoUsuario.js
import { DataTypes } from 'sequelize';
import sequelize from '../config/sequelizeConfig.js';

const TipoUsuario = sequelize.define('TipoUsuario', {
  secuencial: {
    type: DataTypes.BIGINT,
    primaryKey: true,
    autoIncrement: true,
    autoIncrementIdentity: true,
  },
  descripcion: {
    type: DataTypes.STRING(75),
    allowNull: false,
    field: 'descripcion',
  },
  estaActivo: {
    type: DataTypes.SMALLINT, 
    allowNull: false,
    defaultValue: 1,
  },
  // Otros campos de tipo de usuario
});
export default TipoUsuario;